projects/addon-doc/src/components/navigation/navigation.component.ts

```diff
@@ -147,23 +169,63 @@ export class TuiDocNavigationComponent {
         setTimeout(() => {
             this.navigateToAnchorLink(hash);
             this.animateExample(hash);
-        }, SCROLL_INTO_VIEW_DELAY);
+        }, SCROLL_TO_ANCHOR_LINK_DELAY);
     }
 
-    private navigateToAnchorLink(fragment: string) {
-        const element = this.documentRef.querySelector(fragment);
+    private syncNavigationPanel() {
+        this.title$.pipe(first()).subscribe(() => {
```

Comment:
Why is it `first` and not in overall `title$` subscription?

Reply:
related to https://github.com/TinkoffCreditSystems/taiga-ui/pull/254#discussion_r584523469